packages/transport/src/lowlevel/withSharedConnections.ts

```diff
@@ -117,7 +117,7 @@ export default class LowlevelTransportWithSharedConnections {
     defereds: { [id: number]: Deferred<MessageFromSharedWorker> } = {};
     isOutdated = false;
     latestId = 0;
-    name = 'LowlevelTransportWithSharedConnections';
+    name = 'WebUsbTransport';
```

Comment:
I see this `LowlevelTransportWithSharedConnections` was also the name of the class. 
Should the class name stay as it was previously or rename it to `WebUsbTransport`?

Reply:
This file will be removed anyway as I am totally reworking this class in https://github.com/trezor/trezor-suite/pull/6509https://github.com/trezor/trezor-suite/pull/6509https://github.com/trezor/trezor-suite/pull/6509 
I only needed to change this line to make this PR happen